engine/src/image_rep.cpp

```diff
@@ -893,6 +893,45 @@ bool MCImageRepGetFrameDuration(MCImageRep *p_image_rep, uint32_t p_frame, uint3
 	return p_image_rep->GetFrameDuration(p_frame, r_duration);
 }
 
+bool MCImageRepGetMetadata(MCImageRep *p_image_rep, MCArrayRef &r_metadata)
```

Comment:
@livecodeian what do you think about using this in `MCImage::GetMetadataProperty`?

Reply:
I'll take a look but sounds like a good idea to avoid replicating functionality